src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -80,11 +84,19 @@ public function overviewAction()
         @trigger_error(__FUNCTION__ . 'is deprecated since version 8.0 and will be removed in the next major version.', E_USER_DEPRECATED);
 
         return [
-            'is_shop_context' => (new Context())->isShopContext(),
+            'is_shop_context' => $this->get('prestashop.adapter.shop.context')->isShopContext(),
```

Comment:
Why do you sometimes use `$this->get` and sometimes `$this->container->get` ? This should be homogenized, personally I think `$this->get` is more straightforward but maybe there are advantages to use `$this->container->get` that I ignore?